src/commands/createStaticWebApp/BuildPresetListStep.ts

```diff
@@ -12,7 +12,12 @@ import { IStaticWebAppWizardContext } from './IStaticWebAppWizardContext';
 export class BuildPresetListStep extends AzureWizardPromptStep<IStaticWebAppWizardContext> {
     public async prompt(context: IStaticWebAppWizardContext): Promise<void> {
         const placeHolder: string = localize('choosePreset', 'Choose build preset to configure default project structure');
-        const picks: IAzureQuickPickItem<IBuildPreset | undefined>[] = buildPresets.map((pb) => { return { label: pb.displayName, data: pb, group: pb.group === 'ssg' ? localize('ssg', 'Static site generator') : localize('framework', 'Framework') }; });
+
+        // Todo: Remove SSR filter when support is added
+        // https://github.com/microsoft/vscode-azurestaticwebapps/issues/709
+        const picks: IAzureQuickPickItem<IBuildPreset | undefined>[] = buildPresets
+            .filter((pb) => pb.id !== "nextjs")
+            .map((pb) => { return { label: pb.displayName, data: pb, group: pb.group === 'ssg' ? localize('ssg', 'Static site generator') : localize('framework', 'Framework') }; });
```

Comment:
Nit: I think the variable name on these lines should be `bp` short for build preset instead of `pb`. I think we made a typo when this code was written originally.